ofcourse-templates/about.html

```diff
@@ -19,9 +19,11 @@
 </div><!--/span-->
 <div class="col-md-4">
   <h2>Mako</h2>
-  <p><em>"Mako is used by the python.org website, as well as reddit.com where it
-  delivers over one billion page views per month. It is the default template
-  language included with the Pylons and Pyramid web frameworks."</em></p>
```

Comment:
Interestingly, this is also no longer true, mako isn't the pyramid default anymore.